docs/walkthroughs/upgrading-v2-to-v3.md

```diff
@@ -0,0 +1,301 @@
+# **Upgrade from Envoy API v2 to v3**
+
+MARIN3R discovery services support both v2 and v3 API at the same time, which allows for non disruptive upgrades from Envoy v2 API to v3. This walkthrough describes the process of upgrading between API versions.
```

Comment:
```suggestion
MARIN3R discovery services support both v2 and v3 API concurrently, which allows for non-disruptive upgrades from Envoy v2 API to v3. This walkthrough describes the process of upgrading between API versions.
```

Reply:
I think we shoul drop both because `concurrently` is not correct either, as it gives the impression that you can mix v2 and v3 at the same time in the configs, which is not true. What about leaving it like this?

```suggestion
MARIN3R discovery services support both v2 and v3 APIs, which allows for non disruptive upgrades from Envoy v2 API to v3. This walkthrough describes the process of upgrading between API versions.
```

Reply:
Looks good to me!